tests/core/fs/conftest.py

```diff
@@ -145,6 +145,7 @@ def user_fs_offline_state_machine(
     monkeypatch: pytest.MonkeyPatch,
     user_fs_factory: UserFsFactory,
     clear_database_dir: Callable[[bool], None],
+    clear_persistent_mockup: Callable[[], None] | None,
```

Comment:
```suggestion
    clear_persistent_mockup: Callable[[], None],
```
With your fixture, it always return a closure now ;)

Reply:
👍